src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -299,6 +299,17 @@ pub async fn process(request: &Transaction<'_>) -> Result<Response, Error> {
     }
     super::keypath::warn_unusual_keypath(&params, params.name, request.keypath()).await?;
 
+    // Show chain confirmation only for known non-mainnet networks
+    if super::params::is_known_non_mainnet(params.chain_id) {
```

Comment:
why not confirm this unconditionally? Imho it would be good UX to first confirm what we want to do regardless.